src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -279,6 +352,41 @@ public int getColor()
         return color;
     }
 
+    public double getMotionScale(FluidState state, Entity entity)
+    {
+        return motionScale.applyAsDouble(state, entity);
+    }
+
+    public double getFallDistanceModifier(FluidState state, Entity entity)
+    {
+        return fallDistanceModifier.applyAsDouble(state, entity);
+    }
+
+    public boolean canSwim(FluidState state)
+    {
+        return canSwim.test(state);
+    }
+
+    public boolean canDrown(FluidState state, LivingEntity entity)
+    {
+        return canDrown.test(state, entity);
+    }
+
+    public boolean canExtinguish(FluidState state, Entity entity)
+    {
+        return canExtinguish.test(state, entity);
+    }
+
+    public boolean canHydrate(FluidState fluidState, BlockState blockState)
+    {
+        return canHydrate.test(fluidState, blockState);
+    }
+
+    public boolean canBoat(FluidState state, Boat boat)
+    {
+        return this.canBoat.test(state, boat);
+    }
```

Comment:
Part of me feels like it may be useful to add the javadocs to these getters, especially given they are mostly written already on the actual fields. That way it is easier to see the javadocs of what these methods do when using the IDE rather than having to go to implementation and then look at the field and see that it happens to have a javadoc.